### ISSUE TITLE
Session F-Droid repo does not include version 1.16.9

### ISSUE THREAD

**Issue by 2011** [oxen-io/session-android#1244]
**Code of conduct**

- [x] I have read and agree to adhere to the [Code of Conduct](https://github.com/oxen-io/session-android/blob/master/CODE_OF_CONDUCT.md).

**Describe the bug**

I saw version 1.16.9 show up in Izzy's repo (which, much to my annoyance, always includes only the armeami-v7a build - I have a visceral hatred of installing 32-applications on a 64-bit operating system) a couple of days ago, but so far, the Session F-Droid repo does not include that release.

**To reproduce**

Install Session F-Droid repo and sync.

**Smartphone (please complete the following information):**

 - Device: all
 - OS: all
 - Version of Session or latest commit hash: 1.16.9

**Comment by SzBenedek2006** [comment]
Yeah, it's really annoying. I decided to wait, but I couldn't resist and installed the new version from github.
